helper/populate_db.py

```diff
@@ -2,16 +2,9 @@
 import os
 from datetime import datetime, date
 
-from dotenv import load_dotenv
-
 from dbschema.models import Users, ClosedUserAnswers, UserInterventionState, DialogQuestions, DialogAnswers
 from helper import get_db_session
 
-# load .env-file and get db_host and niceday_api_endopint variables
-load_dotenv()
-DB_HOST = os.getenv('DB_HOST', default='db:5432')
-
```

Comment:
Why did you remove loading `dotenv` file? I think it would be nice to allow developers to easily set the db host url.

Reply:
It's because there is no .env on heroku, so everything is read via environment variables. Locally, we either use the docker-compose.yml (which loads the .env file for the container already, meaning anything set in .env is available to `os.getenv()`), or people might launch an individual container with docker, in which case they should use the `--env-file` arg to do the same.

Reply:
Well, you might also just want to run things in your native environment. Then reading in a .env file is necessary (or at least very handy). The nice thing about dotenv is that if there is not .env file, but instead you have environment variables (like in a heroku deploy) it still works :)

Reply:
That's true, but we haven't really supported running it outside of docker for a while now. My worry with this is that it might cause confusion when running with the docker-compose (since it silently reads in a .env whether you've configured docker-compose to use it or not, and might overwrite other vars). But I don't feel so strongly about it that I will refuse to put it back in :)

Reply:
(Hmm and there is a `python-dotenv` package that runs scripts locally and sets the .env as environment that could also maybe used during local dev, so that's maybe a possibility for people who still want to run outside of docker)